notifier.py

```diff
@@ -93,18 +74,19 @@
 print("Running on {}".format(platform))
 if platform == PLT_WIN:
     from win10toast import ToastNotifier
+
     toast = ToastNotifier()
 
 
-def alert(url):
-    product = url_keywords[url][3]
+def alert(site):
+    product = site['name']
     print("{} IN STOCK".format(product))
-    print(url)
-    webbrowser.open(url, new=1)
-    os_notification("{} IN STOCK".format(product), url)
-    sms_notification()
-    discord_notification(product, url)
-    time.sleep(ALERT_DELAY)
+    print(site['url'])
+    webbrowser.open(site['url'], new=1)
+    os_notification("{} IN STOCK".format(product), site['url'])
+    sms_notification(site['url'])
+    discord_notification(product, site['url'])
```

Comment:
use `site.get('myKey')` so missing keys won't cause issues. This will allow us to omit the 'api' key in the .json file which is blank for every site except nvidia at the moment.

Reply:
sorry for the delay, good idea; made the changes